README.md

```diff
@@ -44,12 +44,16 @@ class Restaurant < ActiveRecord::Base
 
   module StyleAdapter
 
-    def self.to_salesforce(value)
-      "#{value} in Salesforce"
+    def self.to(attributes)
```

Comment:
The use of `to` and `from` method names here is nowhere near as clear as the previous naming. :/